test-suite/consensus-tests/tests/basic.rs

```diff
@@ -276,9 +276,9 @@ fn test_store_txs_positions() {
     let snapshot = sandbox.blockchain().snapshot();
     let schema = snapshot.for_core();
     let locations = schema.transactions_locations();
-    for (expected_idx, hash) in hashes.iter().enumerate() {
+    for (expected_idx, hash) in (0u32..).zip(hashes.iter()) {
```

Comment:
Nit: Would `(0_u32..).zip(&hashes)` work here? IMO, this would be more idiomatic.